sections/futures/MarketDetails/MarketDetail.tsx

```diff
@@ -2,48 +2,63 @@ import { ReactElement, memo, FC, useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
 
+import { FlexDivRowCentered } from 'components/layout/flex';
 import { Body } from 'components/Text';
 import Tooltip from 'components/Tooltip/Tooltip';
 import { selectMarketInfo } from 'state/futures/selectors';
 import { useAppSelector } from 'state/hooks';
 
+import HoursToggle from './HoursToggle';
 import { isMarketDataKey, marketDataKeyMap } from './utils';
 
 type MarketDetailProps = {
 	mobile?: boolean;
 	dataKey: string;
 	color?: string;
 	value: string | ReactElement;
+	toggle?: boolean;
```

Comment:
As this is a generic component I think we should make this a generic prop like `extra: ReactElement` where we can render the component outside of here.